contents/docs/integrate/third-party/rudderstack.md

```diff
@@ -5,39 +5,26 @@ sidebar: Docs
 showTitle: true
 ---
 
-> **Note:** The [Official RudderStack Docs page for the PostHog integration](https://www.rudderstack.com/docs/destinations/streaming-destinations/posthog/) provides a better overview of how to set up this integration.
+RudderStack is an open-source, customer data platform for developers. It allows you to collect and deliver customer event data to a variety of destinations across your growth, product, and marketing stack.
 
-## About RudderStack
+> Before integrating with Rudderstack, we recommend you read our [CDP integration guide](/docs/integrate/cdp) to understand the different options for integrating with PostHog.
 
-RudderStack is an open-source, warehouse-first, customer data platform for developers. It allows you to collect and deliver customer event data to a variety of destinations such as data warehouses and analytics platforms.
+## Setting up the Rudderstack Importer
 
-![RudderStack Dashboard](../../../images/rudderstack-dashboard.png)
+Make sure you have a Rudderstack account **and** a PostHog account, using [PostHog Cloud](https://app.posthog.com/signup) or self-hosting.
 
-#### Can PostHog with RudderStack do everything PostHog does by itself?
```

Comment:
I see this section is deleted - do the new instructions mean that Session Recordings, Feature Flags etc will work?  Or do users still need to add the JS Snippet?

Reply:
Yep they work out of the box if they turn on device mode